setup.py

```diff
@@ -48,14 +48,15 @@
     url='https://github.com/ethpm/py-ethpm',
     include_package_data=True,
     install_requires=[
-        'cytoolz==0.9.0',
+        "cytoolz>=0.9.0,<1.0.0;implementation_name=='cpython'",
```

Comment:
to do this you also need to include the `toolz` package for `pypy` installations.